src/main.py

```diff
@@ -36,7 +38,7 @@
     # Load custom extensions
 
     logger.debug("Working directory: " + str(Path.cwd()))
-    extensions = [Path(f).stem for f in glob.iglob("/app/src/extensions/*.py")]
+    extensions = [Path(f).stem for f in glob.iglob("./src/extensions/*.py")]
```

Comment:
Nice catch 👍🏼

Reply:
This might not be needed as the container starts the application from `/app` I think?

Reply:
It's just if the ReadMe tells people to run main.py instead of a container then the bot won't load any extensions.